base/static/base/js/library_website.js

```diff
@@ -17,10 +17,10 @@ function renderEvents() {
             $.each(innerJson, function(i, v){
                 if (v['start_date'] == v['end_date']) {
                     // single-day events.
-                    eventsHtml += '<p><a id="event-header" href="' + v['link'] + '">' + v['title'] + '</a><br/><span class="event-date">' + v['start_date'] + '</span> | ' + v['start_time'] + ' - ' + v['end_time'] + '</p>'
+                    eventsHtml += '<p><a class="event-header" href="' + v['link'] + '">' + v['title'] + '</a><br/><span class="event-date">' + v['start_date'] + '</span> | ' + v['start_time'] + ' - ' + v['end_time'] + '</p>'
                 } else {
                     // multi-day events. 
-                    eventsHtml += '<p><a id="event-header" href="' + v['link'] + '">' + v['title'] + '</a><br/><span class="event-date">' + v['start_date'] + '</span> | ' + v['start_time'] + ' -<br/>' + '<span class="event-date">' + v['end_date'] + '</span> | ' + v['end_time'] + '</p>'
+                    eventsHtml += '<p><a class="event-header" href="' + v['link'] + '">' + v['title'] + '</a><br/><span class="event-date">' + v['start_date'] + '</span> | ' + v['start_time'] + ' -<br/>' + '<span class="event-date">' + v['end_date'] + '</span> | ' + v['end_time'] + '</p>'
```

Comment:
This is a low risk change. This is just the html rendered on the page after the API call. Since you changed the SASS to match, I'm inclined to think this will be OK.